.changeset/large-rockets-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kafka-backend': minor
```

Comment:
dont think this changeset is needed. You only need one changeset per PR normally, and this PR doesn't actually touch the kafka plugin so you can remove this file entirely and just use the `gorgeous-meals-tie`

Reply:
ok, just kept `gorgeous-meals-tie` and removed the rest